src/main/java/com/synopsys/integration/blackduck/imageinspector/image/oci/OciManifestDescriptorParser.java

```diff
@@ -30,32 +30,34 @@ public OciManifestDescriptorParser(ManifestRepoTagMatcher manifestRepoTagMatcher
     }
 
     public OciDescriptor getManifestDescriptor(OciImageIndex ociImageIndex,
-                                               @Nullable String givenRepo, @Nullable String givenTag) throws IntegrationException {
+        @Nullable String givenRepo, @Nullable String givenTag) throws IntegrationException {
         // TODO- Probably also need to select one of multiple based on arch
         List<OciDescriptor> trueManifests =
-                ociImageIndex.getManifests().stream()
-                        .filter(man -> MANIFEST_FILE_MEDIA_TYPE.equals(man.getMediaType()))
-                        .collect(Collectors.toList());
+            ociImageIndex.getManifests().stream()
+                .filter(man -> MANIFEST_FILE_MEDIA_TYPE.equals(man.getMediaType()))
+                .collect(Collectors.toList());
         if (trueManifests.size() == 0) {
             throw new IntegrationException(String.format("No manifest descriptor with media type %s was found in OCI image index", MANIFEST_FILE_MEDIA_TYPE));
         }
-        if ((trueManifests.size() == 1) && StringUtils.isBlank(givenRepo)) {
-            logger.debug(String.format("User did not specify a repo:tag, and there's only one manifest; inspecting that one; digest=%s", trueManifests.get(0).getDigest()));
+        if ((trueManifests.size() == 1)) {
+            logger.debug(String.format("There is only one manifest; inspecting that one; digest=%s", trueManifests.get(0).getDigest()));
```

Comment:
Matching (i.e. choosing the manifest file) should only take place when there are multiple manifests present in the OCI image archive. If there is only 1 manifest, we should select that one and not concern ourselves with reference values and their formats in the index.json (i.e. ignore any given repo/tag value provided by the user for matching since there is only one manifest found)